### DISCUSSION TITLE
为什么demo中公式计算结果不显示？

### DISCUSSION THREAD

**Discussion by bingo618** [dream-num/univer#2154] [Q&A | 常见问题]
<img width="1511" alt="image" src="https://github.com/dream-num/univer/assets/20422686/314ee7a7-ae42-460a-b968-77537fd2b712">

**Comment by jikkai** [comment]
<img width="274" alt="image" src="https://github.com/dream-num/univer/assets/14025786/5da5fac5-b0c7-42d3-86b6-f5f5ffc78bfe">
